_search-plugins/knn/knn-score-script.md

```diff
@@ -313,7 +313,7 @@ A space corresponds to the function used to measure the distance between two poi
   </tr>
   <tr>
     <td>innerproduct</td>
-    <td>\[ Distance(X, Y) = \sum_{i=1}^n (X_i - Y_i) \]</td>
+    <td>\[ Distance(X, Y) = \sum_{i=1}^n (X_i * Y_i) \]</td>
```

Comment:
Should we use the math multiplication sign (dot) instead of the programming multiplication sign (*) here? Or we can omit the multiplication sign altogether because multiplication is implicit in math. 

In general, should we denote vectors as lowercase bold letters and scalars as lowercase non-bold letters? I think this notation is easier to understand and is the default mathematical notation for vectors.

Reply:
Switched to dot. Let's address the 2nd comment in a different issue, since that would require changing more than just this formula.